recipe/meta.yaml

```
@@ -43,47 +43,40 @@ requirements:
   run:
     # appnope should be in orange-widget-base
     - appnope  # [osx]
-    - anyqt >=0.0.13
+    - anyqt >=0.1.0
     - baycomp >=1.0.2
-    - bottleneck >=1.3.2
+    - bottleneck >=1.3.4
     - chardet >=3.0.2
     - httpx >=0.21
-    - joblib >=0.9.4
+    - joblib >=0.11
     - keyring
     - keyrings.alt
-    - matplotlib-base >=2.2.5
+    - matplotlib-base >=3.2.0
     - networkx
-    - numpy >=1.17.3
+    - numpy >=1.19.5
     - openpyxl
     - opentsne >=0.6.1
```

https://github.com/biolab/orange3/blob/e0ee124f82b186de0b693dadf98011ccdf449bf2/requirements-core.txt#L23
```suggestion
    - opentsne >=0.6.1,!=0.7.0
```

---

I'll take it!